robot_driver/include/robot_driver/robot_driver.h

```diff
@@ -301,6 +323,18 @@ class RobotDriver {
   std::vector<double> swing_kp_cart_;
   std::vector<double> swing_kd_cart_;
 
+  /// CF high pass gain
+  std::vector<double> high_pass_a_;
+  std::vector<double> high_pass_b_;
+  std::vector<double> high_pass_c_;
+  std::vector<double> high_pass_d_;
+
+  /// CF low pass gain
+  std::vector<double> low_pass_a_;
+  std::vector<double> low_pass_b_;
+  std::vector<double> low_pass_c_;
+  std::vector<double> low_pass_d_;
```

Comment:
Don't we want these to just be members of the CF class, not robot driver? Someone running a different filter wouldn't need these.